presto-main/src/main/java/com/facebook/presto/server/QueryExecutionResource.java

```diff
@@ -121,7 +121,7 @@ public Response getTaskInfo(@PathParam("queryId") String queryId)
 
                 tasks.add(new Task(taskStatus.getTaskId().toString(),
                         taskStatus.getState().toString(),
-                        taskStatus.getSelf().getHost(),
+                        taskStatus.getSelf().getHost() + ":" + taskStatus.getSelf().getPort(),
```

Comment:
Is this change even necessary? The JS method `getHostname` is called on `task.taskStatus.self` which is a URI.

Where is `Task` used?